ansible/files/ocp/glance/secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: glance-secret
+stringData:
```

Comment:
1 secrets file per service is about as granular as I think we should go. It really gets to be an unwieldy amount of secrets otherwise. Do you have an example where an operator is splitting things out into say 10's of secrets per service?

Per the keystone credentials we might need to add other parameters there over time. For now I took the approach that glance is called 'glance'. We've repeatedly spoken about minimal config options, I think this is definitely an area where we can just have a default set that is always used.